packages/react-styled-core/src/Tooltip/styles.js

```diff
@@ -1,11 +1,16 @@
 import useColorMode from '../useColorMode';
 
-const colorProps = props => {
-  const { variantColor, colorMode } = props;
+const colorProps = ({ variantColor, colorMode }) => {
+  if (variantColor === 'gray') {
+    return {
+      bg: colorMode === 'dark' ? 'gray:10' : 'gray:70',
+      color: colorMode === 'dark' ? 'black:emphasis' : 'white:emphasis',
```

Comment:
Is it supposed to be `black:primary` and `white:primary`?

Reply:
yes, the `black:primary` is correct, fixed in 7a13452, thanks.